api/src/test/java/com/codeforcommunity/rest/HttpConstantsTest.java

```diff
@@ -0,0 +1,19 @@
+package com.codeforcommunity.rest;
+
+// Tests for HttpConstants.java omitted since it just defines constants
+
+/*
+public interface HttpConstants {
+    int client_error_code = 400;
+    int server_error_code = 500;
+    int unauthorized_code = 401;
+    int created_code = 201;
+    int ok_code = 200;
+    String contentType = "Content-Type";
+    String applicationJson = "application/json";
+    String unauthorizedMessage = "unauthorized access for request resources";
+    String clientErrorMessage = "unable to handle request please check request format and consult documentation";
+    String noteIdParam = "note_id";
+    String okMessage = "OK";
+    String noteRoute = "/api/note";
+} */
```

Comment:
Just for consistency's sake, so that the `test` directory completely mirrors `main`.

Reply:
While you want to mirror `main`, you usually want your `test` directory to only contain the classes that either test or help test your classes in the main directory. The mirroring part becomes really useful when you want to test a package default/protected method, since the test directory allows you to have classes be in the same package without actually being in the `main` directory. Does that make sense?